apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/Account.java

```diff
@@ -359,7 +362,7 @@ protected void save(Connection con, AccountProperty accountProperty) throws SQLE
                 publicKeyCache.remove(accountDbKeyFactory.newKey(block.getGeneratorId()));
                 block.getTransactions().forEach(transaction -> {
                     publicKeyCache.remove(accountDbKeyFactory.newKey(transaction.getSenderId()));
-                    if (!transaction.getAppendages(appendix -> (appendix instanceof Appendix.PublicKeyAnnouncement), false).isEmpty()) {
+                    if (!transaction.getAppendages(appendix -> (appendix instanceof PublicKeyAnnouncement), false).isEmpty()) {
```

Comment:
I think, we should add to all Appendix inner classes suffix Appendix

Reply:
> I think, we should add to all Appendix inner classes suffix Appendix
Ok, I will handle that.